test-kuttl/e2e/simple-rsync/10-waitfor-generated-keys.yaml

```diff
@@ -2,5 +2,5 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
-  - timeout: 60
+  - timeout: 1200
```

Comment:
Do we really need this timeout set so high?

Reply:
ACK. Was left from debugging steps. CHanged nack to 60s.